gnmi_cli_py/py_gnmicli.py

```diff
@@ -60,331 +60,360 @@
 
 
 def _parse_path(p_names):
-  """Parses a list of path names for path keys.
-
-  Args:
-    p_names: (list) of path elements, which may include keys.
-
-  Returns:
-    a gnmi_pb2.Path object representing gNMI path elements.
-
-  Raises:
-    XpathError: Unabled to parse the xpath provided.
-  """
-  gnmi_elems = []
-  for word in p_names:
-    word_search = _RE_PATH_COMPONENT.search(word)
-    if not word_search:  # Invalid path specified.
-      raise XpathError('xpath component parse error: %s' % word)
-    if word_search.group('key') is not None:  # A path key was provided.
-      tmp_key = {}
-      for x in re.findall(r'\[([^]]*)\]', word):
-        tmp_key[x.split("=")[0]] = x.split("=")[-1]
-      gnmi_elems.append(gnmi_pb2.PathElem(name=word_search.group(
-          'pname'), key=tmp_key))
-    else:
-      gnmi_elems.append(gnmi_pb2.PathElem(name=word, key={}))
-  return gnmi_pb2.Path(elem=gnmi_elems)
+    """Parses a list of path names for path keys.
+
+    Args:
+        p_names: (list) of path elements, which may include keys.
+
+    Returns:
+        a gnmi_pb2.Path object representing gNMI path elements.
+
+    Raises:
+        XpathError: Unabled to parse the xpath provided.
+    """
+    gnmi_elems = []
+
+    for word in p_names:
+        word_search = _RE_PATH_COMPONENT.search(word)
+        if not word_search:  # Invalid path specified.
+            raise XpathError('xpath component parse error: %s' % word)
+
+        if word_search.group('key') is not None:  # A path key was provided.
+            tmp_key = {}
+
+            for x in re.findall(r'\[([^]]*)\]', word):
+                tmp_key[x.split("=")[0]] = x.split("=")[-1]
+
+            gnmi_elems.append(gnmi_pb2.PathElem(name=word_search.group(
+                              'pname'), key=tmp_key))
+        else:
+            gnmi_elems.append(gnmi_pb2.PathElem(name=word, key={}))
+
+    return gnmi_pb2.Path(elem=gnmi_elems)
 
 
 def _create_stub(creds, target, port, host_override):
-  """Creates a gNMI Stub.
-
-  Args:
-    creds: (object) of gNMI Credentials class used to build the secure channel.
-    target: (str) gNMI Target.
-    port: (str) gNMI Target IP port.
-    host_override: (str) Hostname being overridden for Cert check.
-
-  Returns:
-    a gnmi_pb2_grpc object representing a gNMI Stub.
-  """
-  if creds:
-    if host_override:
-      channel = gnmi_pb2_grpc.grpc.secure_channel(target + ':' + port, creds, ((
-          'grpc.ssl_target_name_override', host_override,),))
+    """Creates a gNMI Stub.
+
+    Args:
+        creds: (object) of gNMI Credentials class used to build the secure channel.
+        target: (str) gNMI Target.
+        port: (str) gNMI Target IP port.
+        host_override: (str) Hostname being overridden for Cert check.
+
+    Returns:
+        a gnmi_pb2_grpc object representing a gNMI Stub.
+    """
+    if creds:
+        if host_override:
+            print("Secure channel with host_override is being created ...")
```

Comment:
>print [](http://example.com/codeflow?start=12&length=5)

Is `print` really needed? Please remove if it is only for your own debug purpose.